### ISSUE TITLE
[Dependency] Migrate From Twit to Twitter-Lite

### ISSUE THREAD

**Issue by tejazz** [tejazz/monocle#5]
Twitter-Lite Repo: https://github.com/draftbit/twitter-lite

**Comment by tejazz** [comment]
Some issues I ran into while trying to add the new twitter-lite library
420 Errors on Restarting a Connection: (Cite: https://github.com/draftbit/twitter-lite/issues/118)
Backing Off Due to Rate Limitation: (Cite: https://github.com/draftbit/twitter-lite/issues/59)

@waynebourne and @jaychaturvedi 
It seems streaming at present on Twitter is highly rate-limited and we keep running into these issues. Do you guys think it would be wiser to move from live streaming to calling a fixed set of data for a `term` or `track` and do our analysis on it? This would help us ship the application to production faster.

**Comment by waynebourne** [comment]
@tejazz , you mean like the top 100 or 200 tweets of the day related to that topic?

**Comment by tejazz** [comment]
Yes. Something around that. Would be very much under the rate limitation.

**Comment by waynebourne** [comment]
That would  be a start. Once the application is live, we can do some digging and explore further options.

**Comment by tejazz** [comment]
Yes. I have a few ideas of how to bypass the rate-limiting. I already reached out to the creator of the library and asked them if they could help. But for now let's focus on making it usable first. @jaychaturvedi Let me know if you have some other ideas.
